faf-commons-data/src/test/java/com/faforever/commons/replay/ZstdUncompressTest.java

```diff
@@ -0,0 +1,35 @@
+package com.faforever.commons.replay;
+
+import org.junit.jupiter.api.Test;
+
+import java.nio.charset.StandardCharsets;
+
+import static com.faforever.commons.test.IsUtilityClassMatcher.isUtilityClass;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class ZstdUncompressTest {
+  public static final byte[] UNCOMPRESSED_BYTES = "fafafafafafafaf".getBytes(StandardCharsets.US_ASCII);
+  public static final byte[] COMPRESSED_BYTES = new byte[]{
+    40, -75, 47, -3, 36, 15, 69, 0, 0, 16, 102, 97, 1, 0, 29, 14, 11, -66, 93, -9, 17
+  };
```

Comment:
make length  it bigger than 4096, pls

Reply:
Covered by reference data decompression test.